lib/std/array_list.zig

```diff
@@ -360,6 +437,15 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
             self.capacity = 0;
         }
 
+        fn computeBetterCapacity(current_capacity: usize) usize {
+            var better_capacity = current_capacity;
+            while (true) {
+                better_capacity +|= better_capacity / 2 + 8;
+                if (better_capacity >= current_capacity) break;
+            }
+            return better_capacity;
+        }
```

Comment:
This could be moved to a top-level function (outside of `ArrayListAligned`/`ArrayListAlignedUnmanaged`) so that both can use the same function (to avoid having two separate implementations of this function).

Reply:
Done.
Ideally, the memory growth policy should be a parameter to (Unmanaged)ArrayList. This would allow settling debates like the one that started [here](https://ziggit.dev/t/problem-with-insertslice-in-arraylist/1779/8). Also, in some cases, users might get better performance by tweaking it.